maestro-macros/src/test/scala/au/com/cba/omnia/maestro/macros/AutomapMacroSpec.scala

```
@@ -19,7 +19,7 @@ import au.com.cba.omnia.maestro.test.Arbitraries._
 import au.com.cba.omnia.maestro.test.thrift.humbug.{Types => HTypes, _}
 import au.com.cba.omnia.maestro.test.thrift.scrooge._
 
-object AutomapSpec extends Spec { def is = s2"""
+object AutomapMacroSpec extends Spec { def is = s2"""
```

The funny thing is we already have a test for that and it was failing however the tests were incorrectly configured (fixed by e56ae8c126ca990afae696644e5ebc3c086a4b7b) so the failure would be ignored and for each test it would only return the result of the last check. Luckily nothing else was caught out by this.

The Automap macro didn't work for transformations with just one rule. This is now fixed.